azure/template.json

```diff
@@ -1,31 +1,35 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
         "serviceName": {
             "type": "string"
         },
-        "aseHostingEnvironmentName": {
-            "type": "string",
-            "defaultValue": ""
-        },
-        "aseResourceGroup": {
-            "type": "string",
-            "defaultValue": ""
-        },
-        "appServicePlanName": {
+        "configurationStorageConnectionString": {
             "type": "string"
         },
-        "appServicePlanResourceGroup": {
+        "environmentName": {
             "type": "string"
         },
-        "configurationStorageConnectionString": {
+        "resourceEnvironmentName": {
             "type": "string"
         },
-        "environmentName": {
+        "sharedBackEndAppServicePlan": {
             "type": "string"
         },
-        "resourceEnvironmentName": {
+        "aspSize": {
+            "type": "string",
+            "defaultValue": "1"
+        },
+        "aspInstances": {
+            "type": "int",
+            "defaultValue": 1
+        },
+        "aspSku": {
+            "type": "string",
+            "defaultValue": "Basic"
+        },
+        "sharedBackEndAppServicePlanResourceGroupName": {
             "type": "string"
         },
         "loggingRedisConnectionString": {
```

Comment:
securestring

Reply:
Changed `loggingRedisConnectionString` to securestring type.